src/libstd/keyword_docs.rs

```diff
@@ -1054,43 +1071,22 @@ mod self_upper_keyword {}
 /// let r2 = &FOO as *const _;
 /// // With a strictly read-only static, references will have the same adress
 /// assert_eq!(r1, r2);
+/// // A static item is used just like a variable
```

Comment:
Tiny nit, but "*is* just like a variable" is technically wrong. Maybe just write "Print the static value" or something like that?

Reply:
Maybe "can be used just like a variable in many case" ? Writing "Print the static value" feels redundant

Reply:
So the full text would be: `A static item can be used just like a variable in many cases`